common/components/filter/_filter.scss

```diff
@@ -0,0 +1,76 @@
+.app-filter {
+  margin: 0 0 govuk-spacing(3);
+}
+
+.app-filter__list {
+  list-style: none;
+  margin: 0;
+  padding-left: 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: stretch;
+
+  flex-wrap:wrap;
+  flex-direction:row;
+
+  @include govuk-font($size: 19, $weight: normal);
+  @include govuk-media-query($until: 370px) {
+    flex-direction:column;
+  }
+  @include govuk-media-query($media-type: print) {
+    display: none;
+  }
+}
+
+.app-filter__list-item {
+  border: 2px solid govuk-colour('blue');
+  flex: 1 1 0;
+  min-height: 70px;
+  display: flex;
+  @include govuk-media-query($until: 370px) {
+    min-height: 90px;
+  }
+}
+
+.app-filter__list-item--active {
+  background: govuk-colour('white');
+  border: 2px solid govuk-colour('black');
+}
+
+.app-filter__link {
+  width: 100%;
+  height: 100%;
+  // cannot remove text-decoration from parts of the link, so we remove it everywhere and re-add it when necessary
+  text-decoration: none;
+  background: govuk-colour('blue');
+  color: govuk-colour('white');
+  display: block;
+  flex-grow: 1;
+
+  &:link,
+  &:active,
+  &:visited {
+    color: govuk-colour('white');
+  }
+
+  &:hover {
+    color: govuk-colour('light-grey');
+  }
+
+  &:focus {
```

Comment:
![image](https://user-images.githubusercontent.com/3327997/77419806-e9177b00-6dc0-11ea-9c43-f99d4db06453.png)

We might want to set the colour to black in this state. The white on yellow is hard to read.